README.md

```diff
@@ -0,0 +1,219 @@
+
+- Staying late in the office; I usually head home by 6pm
+- Working on weekends
+- Using email or Slack off-hours
+- Scheduling meetings very early or very late in the day
+
+Given that I try to not do these myself, it would be unfair of me to expect you to. If you find me not leading by example --by exhibiting the above (or other) behaviors or encouraging behavior from you that leads to poor balance of work with life-- please voice your objection. The most likely explanation is that I did not consciously realize I was doing this and speaking up will help me to correct course.
+
+I prefer people to work smarter and not harder. If you find that you're needing to work well beyond 40 hours to be successful in your role, we should have a conversation about what adjustments can be made to make this not the case.
+
+_*The obvious exception here is emergency situations._
+
+## Communicating With Me
+
+My primary guideline for communication is to consider what channel is most appropriate for our interaction. If you're looking for more than five minutes of back-and-forth, let's talk in person. If you have a quick question, Slack is great. Just trying to disseminate some non-urgent information to a broad group? Email seems like a solid choice. Here's a few other details about how I operate with each channel:
+
+### Talking
+
+I'm happy to make time to discuss something in-person whenever you need to, whether that's scheduled via a meeting or ad-hoc. For the latter, a heads up on Slack is cool or just stop by and give me a good old-fashioned tap on the shoulder. If I've got my headphones on it typically means I'm focused on something but that should never stop you from interrupting me. Again, you're my first priority.
+
+### Slack
+
+Generally, the easiest way to get in touch with me. I try to stay on top of Slack and respond in less than an hour during the workday. The one complication is that I'm frequently in meetings. It's my goal to be focused in a meeting and not doing other things. I typically play catch up on Slack between meetings. In the event that I'll need more time to respond, I will try to let you know that.
+
+### Email
+
+I'm quite averse to it and try to spend as little time as possible dealing with it. Typically, I'll lump my email processing into a morning session and an afternoon session. However, if the day is full, this can slip. When something is urgent, please do not use email or we're both going to have a sad time.
+
+### PagerDuty
+
+If it's off-hours and urgent, please page me.
+
+### Mobile phone
+
+I generally try to avoid disseminating my mobile number at work. If you think you'll need it, let's talk.
+
+### Social media
+Nope.
```

Comment:
lol. +1 to this in whatever way you meant it. I'm assuming this means "I have no interest in sharing this part of our lives", which makes sense. Maybe good to be clear as to why you feel that way?

Reply:
Well, I don't have any active social media accounts. So, this really just means 'nope' in the simplest form. I could expand on this but 'nope' gives me a chuckle. It might give others a chuckle as well and I don't want to deprive anyone of chuckles.